extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/ValidationParserHookBuildItem.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.qute.deployment;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+
+import io.quarkus.builder.item.MultiBuildItem;
+import io.quarkus.qute.ParserHelper;
+
+/**
+ * This build item can be used to hook into the parser logic during validation at build time.
+ * <p>
+ * Validation parser hooks are never used at runtime.
+ */
+public final class ValidationParserHookBuildItem extends MultiBuildItem {
```

Comment:
@ia3andy This one will be needed to implement https://github.com/quarkiverse/quarkus-qute-web/issues/96.